lib/.editorconfig

```diff
@@ -149,6 +149,10 @@ csharp_space_between_square_brackets = false
 dotnet_code_quality.ca1802.api_surface = private, internal
 
 # C++ Files
+
+# CS1572: XML comment has a param tag, but there is no parameter by that name
+dotnet_diagnostic.CS1572.severity = none

```

Comment:
Why do we need this?

Reply:
VS suggested this. I thought of suppressing the warning but didn't do. Should I remove this?